src/QA/QA.Web/ViewModels/VacancyReviewSearchResultViewModel.cs

```diff
@@ -23,7 +23,7 @@ public class VacancyReviewSearchResultViewModel
 
         public bool IsClosed { get; set; }
 
-        public bool IsNotAvailableForReview => !IsAvailableForReview;
+        public bool IsNotAvailableForReview => !IsAvailableForReview && !IsClosed;
```

Comment:
IsNotAvailableForReview should be just be a simple inverse of IsAvailableForReview.  We shouldn't add extra bits to it as it'll no longer be the case.  Whatever is setting IsAvailableForReview needs to take into account if the review is closed.